aten/src/ATen/DeviceAccelerator.cpp

```diff
@@ -45,6 +45,7 @@ std::optional<c10::DeviceType> getAccelerator(bool checked) {
   DETECT_AND_ASSIGN_ACCELERATOR_COMP(HIP)
   DETECT_AND_ASSIGN_ACCELERATOR_COMP(MPS)
   DETECT_AND_ASSIGN_ACCELERATOR_COMP(HPU)
+  DETECT_AND_ASSIGN_ACCELERATOR_COMP(XLA)
```

Comment:
Adding xla here has quite deep implications FYI. It will change the behavior of many higher level systems wrt the xla device.